lib/utils/get-registry-ky.ts

```diff
@@ -1,29 +1,47 @@
-import ky from "ky"
+import ky from "ky";
 
-// Flag to identify if we're running inside runframe specifically
+// Flag to identify environment configuration
 declare global {
   interface Window {
-    __RUNFRAME_REGISTRY_BASE_URL__?: boolean
+    __RUNFRAME_REGISTRY_BASE_URL__?: boolean;
+    REGISTRY_API_BASE_URL?: string; // For custom API URL
```

Comment:
move to a central location for the types e.g. `lib/window-globals.ts`, also comment on this existing type with a docstring (it should be `FILE_SERVER_API_BASE_URL`

https://github.com/tscircuit/runframe/blob/f6ebd8c4f0f6f7f21fa0f7294b6737c605fea07f/lib/components/RunFrameWithApi/types.ts#L109

Reply:
also we might want to have a prefix `TSCIRCUIT_REGISTRY_API_BASE_URL` because it's in the global scope

Reply:
likewise for the file server, `TSCIRCUIT_FILE_SERVER_API_BASE_URL`